manifest

```diff
@@ -10273,7 +10273,7 @@ f usr/lib/mdb/proc/amd64/libavl.so 0555 root sys
 f usr/lib/mdb/proc/amd64/libc.so 0555 root sys
 f usr/lib/mdb/proc/amd64/libnvpair.so 0555 root sys
 f usr/lib/mdb/proc/amd64/libproc.so 0555 root sys
-f usr/lib/mdb/proc/amd64/libpython3.9.so 0555 root sys
+f usr/lib/mdb/proc/amd64/libpython3.12.so 0555 root sys
```

Comment:
This confuses me slightly... Is this library coming from pkgsrc rather than e.g. illumos-extra? Why does this change happen here?

Reply:
For python, yes.  Only onbld tools use it, so it's exepect to only be needed debugging onbld tools in a build-smartos zone.